lib/os/heap.h

```diff
@@ -139,26 +143,6 @@ static inline chunksz_t chunk_size(struct z_heap *h, chunkid_t c)
 	return chunk_field(h, c, SIZE_AND_USED) >> 1;
 }
 
-static inline void set_chunk_used(struct z_heap *h, chunkid_t c, bool used)
```

Comment:
Why did this function move?  Seems like a needless change.

Reply:
because in this function, my change used other functions like chunksz_to_bytes and solo_free_header, which are declared and defined after this set_chunk_used.